src/Sarif/Visitors/SarifCurrentToVersionOneVisitor.cs

```diff
@@ -57,7 +57,7 @@ internal AnnotatedCodeLocationVersionOne CreateAnnotatedCodeLocationVersionOne(L
                 annotatedCodeLocation = new AnnotatedCodeLocationVersionOne
                 {
                     Annotations = v2Location.Annotations?.Select(CreateAnnotationVersionOne).ToList(),
-                    FullyQualifiedLogicalName = v2Location.LogicalLocation?.FullyQualifiedName,
+                    FullyQualifiedLogicalName = v2Location.LogicalLocations?[0]?.FullyQualifiedName,
```

Comment:
>LogicalLocations [](start = 59, length = 16)

this change shouldn't have been required. why was it?

Reply:
u r right! its not needed!!!!!

---
In reply to: [285815773](https://github.com/microsoft/sarif-sdk/pull/1494#discussion_r285815773) [](ancestors = 285815773)

Reply:
i have reverted the code

---
In reply to: [285824244](https://github.com/microsoft/sarif-sdk/pull/1494#discussion_r285824244) [](ancestors = 285824244,285815773)